.github/workflows/ci.yml

```diff
@@ -32,8 +32,6 @@ jobs:
       # Run the NPM install command before proceeding
       - run: npm install
 
-      - run: npm run lint
```

Comment:
Consistently failing and unnecessary to run on build

Reply:
Everything looks good